jcl/src/java.base/share/classes/module-info.java.extra

```diff
@@ -61,12 +61,14 @@ uses com.ibm.gpu.spi.GPUAssist.Provider;
 exports com.ibm.gpu.spi to
     openj9.gpu;
 /*[IF CRIU_SUPPORT]*/
-exports openj9.internal.criu to
+exports com.ibm.security.criu to
```

Comment:
In hindsight, I should have asked in ibmruntimes/openj9-openjdk-jdk11#510 why it uses `com.ibm`.

Reply:
@ZainabF92 were you able to open PRs for this

Reply:
Yes, please see [PR#521](https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/521)

Reply:
See https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/521